packages/swing-store/src/transcriptStore.js

```diff
@@ -80,15 +80,17 @@ export function makeTranscriptStore(
     )
   `);
 
-  // Transcripts are broken up into "spans", delimited by heap snapshots.  If we
-  // take heap snapshots after deliveries 100 and 200, and have not yet
-  // performed delivery 201, we'll have two non-current (i.e., isCurrent=null)
-  // spans (one with startPos=0, endPos=100, the second with startPos=100,
-  // endPos=200), and a single empty isCurrent==1 span with startPos=200 and
-  // endPos=200.  After we perform delivery 201, the single isCurrent=1 span
-  // will will still have startPos=200 but will now have endPos=201.  For every
-  // vatID, there will be exactly one isCurrent=1 span, and zero or more
-  // non-current (historical) spans.
+  // Transcripts are broken up into "spans", delimited by heap snapshots.
+  // For every vatID, there will be exactly one current span with isCurrent=1,
+  // and zero or more non-current (historical) spans with isCurrent=null.
+  // If we take a heap snapshot after the first hundred deliveries and again
+  // after the second hundred (i.e., after zero-indexed deliveries 99 and 199),
+  // and have not yet performed a delivery after the second snapshot, we'll have
+  // two historical spans (one with startPos=0 and endPos=100, the second with
+  // startPos=100 and endPos=200) and a single empty current span with
+  // startPos=200 and endPos=200.  After we perform the next delivery, the
+  // single current span will still have startPos=200 but will now have
+  // endPos=201.
```

Comment:
@warner I updated this to avoid mixing zero-indexed and one-indexed values; please look it over for correctness.

Reply:
Yeah, I think that's correct. Notes:
* "exactly one current span with isCurrent=1" admits the thought that maybe some "current spans" *don't* have isCurrent=1. Maybe say "exactly one current span (i.e. isCurrent=1)" ? Ditto for the second clause.
* Yep, my previous wording was confused, if the first span included a delivery=100, then it should have had endPos=101, because we increment the span record's `.endPos` just after adding the item.